apps/vr-tests/screener.config.js

```diff
@@ -4,7 +4,8 @@ module.exports = {
   apiKey: process.env.SCREENER_API_KEY,
   resolution: '1024x768',
   baseBranch: (process.env.TRAVIS_PULL_REQUEST !== 'false' && process.env.TRAVIS_BRANCH) || 'master',
-  failureExitCode: 0
+  failureExitCode: 0,
+  cssAnimations: true
```

Comment:
Is this change required? We're seeing changes in components that use animation (like spinner and teaching bubble). With animation set to true we might see these components flicker all the time. Can this be turned on for a single test rather than all of them?

Reply:
If I pass 'cssAnimations' as true then I don't get the border while dragging in the snapshot, because it fades away. But on passing it as false (or not specifying it) the border persists and as a result the drop hint and the border appear together. So although having the border makes the screener test more robust, but functionally is a bit misleading. So shall I keep the config file unchanged( i.e animation set to false)
The drop hint and the border can be shown independently in different snapshots if we can have different value of cssAnimations in the config file for the 2 snaphots.
I am not sure how/if we can modify configs for a single test/different steps within the same test.

Reply:
screener team is looking at implementing settings per story (so that you can turn animations on when you need them). Please remove cssAnimations for now, and we'll turn i back on once we can do it for just this one test.

Reply:
Alright. Removed.